components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/resources/claim-management.yaml

```
@@ -185,6 +185,113 @@ paths:
         500:
           $ref: '#/responses/ServerError'
 
+  /claim-dialects/{dialect-id}/claims/file/{claim-id}:
```

Let's use the path
```suggestion
  /claim-dialects/{dialect-id}/claims/{claim-id}/file:
```
since the response will contain configs of a single claim.